conanfile.py

```diff
@@ -44,7 +44,6 @@ def requirements(self):
         self.requires("viam-cpp-sdk/0.7.0")
         self.requires("tensorflow-lite/2.15.0")
         self.requires("abseil/20240116.2", override=True)
-        self.requires("xtensor/0.25.0")
```

Comment:
...I had assumed this was here because it was important/needed. but I guess I can't argue with results: if it works, it works.

Reply:
It was probably needed, but only because of a flaw in the C++ SDK conan setup: https://github.com/viamrobotics/viam-cpp-sdk/commit/1b0f4a8d533d347d51b268bd443bd70b2819eadd#diff-63f09721ade419d29886e8190ffa3c9c756baa9bef8802a7f2aeabf3b545164fR52. It no longer is with that fix.